src/components/Chat/AnonymousInputMessage.tsx

```diff
@@ -75,9 +67,10 @@ const styles = StyleSheet.create({
 
 export interface AnonymousInputMessageProps {
   onSendButtonClicked: (message: string) => void;
+  type: 'SIGNED' | 'ANONYMOUS';
 }
 
-const AnonymousInputMessage = ({onSendButtonClicked}: AnonymousInputMessageProps) => {
+const AnonymousInputMessage = ({onSendButtonClicked, type}: AnonymousInputMessageProps) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `type` prop allows for more flexibility in how the component is used. However, ensure that all instances where this component is used have been updated to pass this new prop.
```diff
- export interface AnonymousInputMessageProps {
-   onSendButtonClicked: (message: string) => void;
- }
+ export interface AnonymousInputMessageProps {
+   onSendButtonClicked: (message: string) => void;
+   type: 'SIGNED' | 'ANONYMOUS';
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->